### PR TITLE
techport scraped data update

### PR DESCRIPTION
**Pull request by aarontrmartin**
changed load data in s3 bucket to one with capitalized column names, final product should be the same still though, added code to the rest

### CODE REVIEW

**Review by aarontrmartin — COMMENTED** [review]
imported minmaxscaler from scikit-learn (sklearn), wasnt working for me but the code worked when it ran in my global env

**Comment by rwxhuang** [comment]
Code looks good to me, just clean-up would be nice to make it more readable. For example, all I want in def load_processed_data() is the following:

    df = encode_locations(df) \\
    df = encode_status(df) \\
    df = normalize_views(df, 0, 1)
       
The method functions can go in a data_utils.py file (which you'll make and make sure they well-commented). Just import these functions into data_collection.py with `from utils.data_utils import encode_locations, [etc.]`. After this I might nitpick some stuff, but overall logic is great.
